app/src/main/res/xml/grayscale_icon_map.xml

```diff
@@ -126,8 +128,6 @@
     <icon drawable="@drawable/calendar" package="com.samsung.android.calendar" name="Calendar" />
     <icon drawable="@drawable/calendar" package="foundation.e.calendar" name="Calendar" />
     <icon drawable="@drawable/calendar" package="com.appgenix.bizcal" name="Calendar" />
-    <icon drawable="@drawable/calendar" package="org.lineageos.etar" name="Calendar" />
```

Comment:
Should this line be added back but linked to the Etar icon or is it an invalid package name?